portal/src/PhoneTextField.tsx

```diff
@@ -12,7 +12,7 @@ export interface PhoneTextFieldProps {
   pinnedList?: string[];
   allowlist?: string[];
   inputValue: string;
-  onChange: (validValue: string, inputValue: string) => void;
+  onChange: (validValue: string, inputValue: string, dialCode?: string) => void;
```

Comment:
No need to add dialCode. `validValue` is already the E.164 value we want.

Revert the changes in this file.